content/blog/vscode-esp-idf-v1.8.0/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "ESP-IDF Visual Studio Code Extension v1.8.0 release"
+date: 2024-07-12T16:07:09+08:00
+showAuthor: false
+tags: ["News", "ESP-IDF extension", "VSCode", "Visual Studio Code"]
+authors:
+  - "brian-ignacio"
+---
+
+## Introduction
+
+[ESP-IDF Visual Studio Code Extension](https://github.com/espressif/vscode-esp-idf-extension) v1.8.0 has been released! It introduces new features and many improvements. If you face any issue or improvement ideas please share them as a [Github issue](https://github.com/espressif/vscode-esp-idf-extension/issues).
+
+Key features;
+
+- **Eclipse CDT GDB Debug Adapter**
+
+- **ESP-IDF version switcher**
+
+- **Code Navigation without full project build**
+
+- **ESP-IDF Hints viewer**
+
+For other updates please review [v1.8.0 Release notes](https://github.com/espressif/vscode-esp-idf-extension/releases/tag/v1.8.0).
+
```

Comment:
Still suggest adding this to keep the momentum:

Now let's look into these features one by one.